Source/CommandBar/CommandBar.html

```diff
@@ -7,6 +7,20 @@
 
 <template>
     <require from="./CommandBar.scss"></require>
-
-    <slot></slot>
+    <nav class="command-bar ${labelPosition ? 'label-' + labelPosition : ''}">
+        <div class="footprint ${commandLocation ? 'commands-'+ commandLocation : ''}">
+            <template part="content-block" replaceable> </template>
+            <template part="primary-commands" replaceable> </template>
+            <div class="secondary-commands-wrapper ${displaySecondaryCommands ? 'visible' : ''}">
+                <template part="secondary-commands" replaceable></template>
+                <a href="#" class="secondary-commands-toggle" click.delegate="toggleSecondaryCommands()">
+                    <svg width="40" height="24" viewBox="0 0 40 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Not sure what this icon is; but could we have it extracted somehow? HTML gets cluttered when SVGs are embedded directly in it.

Reply:
Yes, we have separate tasks for SVGs and don't think that we should spend time in this task to solve that. Since we have minor issues with files and reffering svgs across components and apps, the easiest for this task was to inline them for now, and fix it in the dedicated SVG task